build/cloudbuild.yaml

```diff
@@ -0,0 +1,17 @@
+# Build the module.
+steps:
+- name: gcr.io/cloud-builders/go:debian
```

Comment:
Do we want to use the k8s build images that have specific versions of golang, like in https://github.com/kubernetes-csi/csi-release-tools/pull/87/files#diff-e5fc65747fb9417968a3295dc1ac5111R26?

Reply:
I think go:debian get the latest version of go https://github.com/GoogleCloudPlatform/cloud-builders/pull/132. Based on that PR's comment, seems like it also does not use specific version of go. @pohly might help on this?

Reply:
Please have a look at https://github.com/kubernetes-csi/csi-release-tools/issues/86#issuecomment-634509901 an the PRs linked to there.

In a nutshell, the approach there is to install the right version of the Go toolchain before using it, using the normal build rules for Windows binaries.

Reply:
The relevant code is https://github.com/pohly/csi-release-tools/blob/bd416901d4b7cad7bc70a39f508b68e2a9fff1f6/cloudbuild.sh which sources `release-tools/prow.sh` and then invokes image building with `run_with_go "${CSI_PROW_GO_VERSION_BUILD}" make push-multiarch`.

You could do something similar and use `run_with_go "${CSI_PROW_GO_VERSION_BUILD}" make build` to produce the csi-proxy.exe.
```

Reply:
got it. Thanks!
I updated it.